packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1795,15 +1795,6 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   @override
   TextEditingValue get currentTextEditingValue => _value;
 
-  @override
-  void updateEditingValueWithDeltas(List<TextEditingDelta> textEditingDeltas) {
-    TextEditingValue value = _value;
-    for (final TextEditingDelta delta in textEditingDeltas) {
-      value = delta.apply(value);
-    }
-    updateEditingValue(value);
-  }
```

Comment:
Since we're losing this method, should we include it as an example implementation of updateEditingValueWithDeltas in the docs or something?  I want to make sure that users know what to do here when they implement updateEditingValueWithDeltas.

Reply:
That makes sense to me!